docs/measurements/procedures/rigid_waist_shift.md

```diff
@@ -12,10 +12,63 @@
       After global corrections are done and trimmed in the machine, one applies a rigid waist shift in a given IR and scans the colinearity knob for the value that minimizes the $|C^{-}|$.
       These settings, when taking away the rigid waist shift, will minimize local coupling and its impact at the IP.
 
+## Rigid Waist Shift & LSA Knobs
+
+Some information about the rigid waist shift can be found in [this gallery page][rws_gallery]{target=_blank} from Felix's package's documentation.
+Some details about the creation of these knobs and their addition in LSA can be found in this [LHC OMC logbook entry][logbook_entry]{target=_blank} and the ones that follow.
+
+### Knob Setting Convention
+
+Two different knobs are used for the waist shift that need to be trimmed in: 
+
+- The triplets knob, which generates the waist shift itself.
+- The independent quadrupoles knob (Q4 - Q10), which re-matches the optics.
+
+The unit setting of the triplet knob is an arbitrary definition, where a value of $\pm1$ corresponds to a $\pm 0.5$% change of the triplet powering, which leads to a waist shift of ~43-44cm in a given direction.
+The knob setting is defined as follows, with regards to Beam1 (reverse for Beam2):
+
+- Unit setting of +$1$: shifts the waist to the _left_ of the IP ($s_{waist}^{B1} < s_{IP}^{B1}$)
+- Unit setting of -$1$: shifts the waist to the _right_ of the IP ($s_{waist}^{B1} > s_{IP}^{B1}$)
+
+### The Knobs in LSA
+
+Since the triplet knob for a unit setting of +$1$ is the exact opposite magnet powering of the knob for a unit setting of -$1$, only a single triplet knob per IP has been added to LSA, which can then be trimmed with a factor of $\pm1$.
+On the other hand, the re-matching quadrupole knobs are specific to a given direction of the shift, so two of them per IP (one per direction) were added to LSA, with the *pos* and *neg* suffixes.
+
+As a consequence, respect the following rules when applying the knobs below:
+
+ - When trimming the triplet knob with a factor of +$1$, use the *pos* re-matching quadrupoles knob.
+ - When trimming the triplet knob with a factor of -$1$, use the *neg* re-matching quadrupoles knob.
+
```

Comment:
should *neg* knob be trimmed in with +1 or -1 when trimming the RWS to -1?

Reply:
and  ... sign convention the same for B2 I assume? maybe two examples, one simple for B1 one with *pos* and the most complex one for B2 with *neg* could help

Reply:
It is correct as written. Trim the triplets with +1 -> trim the quads with *pos* (and the other way around)

Reply:
so, RWS knob to -1, then setting the *neg* quad knob also to -1? maybe I am misreading something here

Reply:
> so, RWS knob to -1, then setting the _neg_ quad knob also to -1? maybe I am misreading something here

Ah I see what you mean. No the quads one should always be trimmed at +1, there is no sign switch as the solution is degenerate. I will probably add an example!

Reply:
Example added in [50dc511](https://github.com/pylhc/pylhc.github.io/pull/74/commits/50dc511ad1c25076db04f1504c12adcfc1a4ed82)